pmultiqc/modules/common/file_utils.py

```
@@ -104,3 +104,13 @@ def get_clean_stem(path):
         if name.endswith(ext):
             return name[: -len(ext)]
     return Path(path).stem
+
+
+def file_prefix(path):
+    try:
+        path = os.path.normpath(path)
+        if "\\" in path:
+            path = path.replace("\\", "/")
+        return Path(path).stem
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)